### DISCUSSION TITLE
Pls, add different speakers

### DISCUSSION THREAD

**Discussion by SlimRG** [openai/whisper#958] [Ideas]
If video has different speakers I want to know, where Speaker1 or where Speaker2.
I use AI TTS after your app.
So, it will be very helpfull

**Comment by SlimRG** [comment]
Smth, like
193
00:15:44,520 --> 00:15:45,520
Speaker2: That's right.

**Comment by phineas-pta** [comment]
unfortunately you have to do it yourself

search for "diarization" in discussions

**Comment by Majdoddin** [comment]
[www.lexicaps.com](http://www.lexicaps.com/) seamlessly adds diarization to Whispers transcription. No 3rd party packages.
Feel free to contact me if you need more features.
Announcement: https://github.com/openai/whisper/discussions/1537
Repo: https://github.com/Majdoddin/lexicaps
